config.yml

```diff
@@ -3,7 +3,7 @@ project: "IN-CORE Lab"
 author: "Yong Wook Kim"
 Channel: "in-core"
 # version form must be "x.x.x"
-INCORE_VERSION: "5.1.0"
-PyIncore: "1.17.0"
+INCORE_VERSION: "5.3.0"
+PyIncore: "1.18.0"
```

Comment:
We haven't have 1.18.0 yet. Maybe you can make it after it gets released. Just a suggestion